src/figma/darkTheme.json

```diff
@@ -13,31 +13,31 @@
       "description": "For secondary neutral backgrounds."
     },
     "default-hover": {
-      "value": "#2d3034",
+      "value": "#282b2e",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
       "description": "For \"hover\" states that use background-default."
     },
     "default-pressed": {
-      "value": "#363b3f",
+      "value": "#36383b",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
       "description": "For \"pressed\" states that use background-alternative."
     },
     "alternative-hover": {
-      "value": "#1e2124",
+      "value": "#191b1d",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
       "description": "For \"hover\" states that use background-alternative."
     },
     "alternative-pressed": {
-      "value": "#272b2f",
+      "value": "#27292a",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
       "description": "For \"pressed\" states that use background-alternative."
     },
     "hover": {
-      "value": "#ffffff0a",
+      "value": "#ffffff05",
```

Comment:
Matches what the current colors are https://github.com/MetaMask/design-tokens/blob/e74e09f72f6415ad5b90ec5bb40498fede1a8cba/src/figma/tokens.json#L1259-L1300